lmnet/lmnet/datasets/dataset_iterator.py

```diff
@@ -306,6 +306,12 @@ def get_shuffle_index(self):
         self.seed += 1
         return random_indices
 
+    def close_dataset(self):
```

Comment:
Since what is closed here is trivial, how about changing the method name to `close`?

Reply:
yep, that's better name, thank you